lib/projects/localDev/DevServerManagerV2.ts

```diff
@@ -41,54 +52,38 @@ class DevServerManagerV2 {
     await Promise.all(this.devServers.map(devServer => callback(devServer)));
   }
 
-  async setup({
```

Comment:
These lifecycle methods don't need to accept arguments anymore, since they use the `LocalDevState`